roles/ngi_pipeline/tasks/main.yml

```diff
@@ -134,7 +134,7 @@
 
 - name: Add py3 anaconda env path to sthlm sourceme script
   lineinfile: dest={{ ngi_pipeline_conf }}/{{ bash_env_sthlm_script }}
-              line='export PATH={{ anaconda_path }}/envs/{{ NGI_venv_py3_name }}/bin:$PATH'
+              line='export PATH={{ anaconda_path }}/envs/{{ NGI_venv_name }}/bin:$PATH'
```

Comment:
We don't really need to keep this if the py3 env is the main env, but no harm in leaving it in, I guess.

Reply:
Ah, no you're right, let's remove it 👍